Neos.Neos/Classes/Package.php

```diff
@@ -76,6 +76,7 @@ public function boot(Bootstrap $bootstrap)
         $dispatcher->connect(Node::class, 'nodeRemoved', ContentCacheFlusher::class, 'registerNodeChange');
         $dispatcher->connect(Node::class, 'beforeNodeMove', ContentCacheFlusher::class, 'registerNodeChange');
 
+        $dispatcher->connect(AssetService::class, 'assetUpdated', ContentCacheFlusher::class, 'registerAssetResourceChange');
```

Comment:
Well, the slot is named `registerAssetResourceChange`, which in this case is not what happens. I'd like to check if we need a new slot or should maybe rename the existing slot.